app/src/main/java/com/example/kuba10/mypokemonplaces/PlaceDetailsFragment/PlaceDetailsFragment.java

```diff
@@ -1,180 +0,0 @@
-package com.example.kuba10.mypokemonplaces.PlaceDetailsFragment;
-
-import android.app.Dialog;
-import android.graphics.Color;
-import android.graphics.drawable.ColorDrawable;
-import android.os.Bundle;
-import android.support.v4.app.DialogFragment;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.ImageButton;
-import android.widget.ImageView;
-import android.widget.TextView;
-
-import com.example.kuba10.mypokemonplaces.Constants;
-import com.example.kuba10.mypokemonplaces.ListFragment.FirebaseListFragment;
-import com.example.kuba10.mypokemonplaces.Model.PokePlace;
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.example.kuba10.mypokemonplaces.R;
-import com.google.firebase.database.DatabaseReference;
-import com.google.firebase.database.Query;
-import com.squareup.picasso.Picasso;
-
-import java.util.ArrayList;
-
-public class PlaceDetailsFragment extends DialogFragment {
-
-
-    private TextView title, description;
-    public ImageView image, showLocation;
-    public ImageButton favouriteBtn;
-
-    private PokePlace userSelectedPlace;
-    private ArrayList<PokemonGo> pokemonGo_data_list;
-    private FirebaseListFragment parentFragment;
-    private Query query;
-
-    public static PlaceDetailsFragment newInstance(PokePlace place) {
-
-        PlaceDetailsFragment fragment = new PlaceDetailsFragment();
-        Bundle args = new Bundle();
-        args.putParcelable(Constants.DETAIL_POKEMON, place);
-        fragment.setArguments(args);
-
-        return fragment;
-    }
-
-    @Override
-    public void onStart() {
-        super.onStart();
-        setFullScreenStyle();
-    }
-
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        if (getArguments() != null) {
-
-            userSelectedPlace = getArguments().getParcelable(Constants.DETAIL_POKEMON);
-            parentFragment = (FirebaseListFragment) getFragmentManager().findFragmentByTag(Constants.LIST_FRAGMENT_TAG);
-            query = parentFragment.getQuery();
-        }
-    }
-
-    @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-                             Bundle savedInstanceState) {
-
-
-        View view = inflater.inflate(R.layout.fragment_place_details, container, false);
-
-        pokemonGo_data_list = parentFragment.sendPokemonListToAdapter();
```

Comment:
Co to robi?

Reply:
Przekauje liste pokemonow do adaptera - ponownie jakos utknalem z dystrybucja danych miedzy klasa ktora je sciaga, klasa glowan fragmentami i adapterami - i pewnie po to dagger jest....